frontend/src/metabase/public/containers/PublicDashboard/PublicDashboard.tsx

```diff
@@ -244,14 +244,19 @@ class PublicDashboardInner extends Component<PublicDashboardProps> {
         setParameterValueToDefault={setParameterValueToDefault}
         enableParameterRequiredBehavior
         actionButtons={
-          buttons.length > 0 && <div className={CS.flex}>{buttons}</div>
+          buttons.length > 0 ? <div className={CS.flex}>{buttons}</div> : null
         }
         dashboardTabs={
           dashboard?.tabs &&
           dashboard.tabs.length > 1 && (
             <DashboardTabs dashboardId={this.props.dashboardId} />
           )
         }
+        bordered={bordered}
```

Comment:
We can now pass these display props through to the EmbedFrame, removing its reliance on `location`

Reply:
Nice!!!